defold-spine/pluginsrc/com/defold/bob/pipeline/SpineSceneBuilder.java

```diff
@@ -32,4 +36,27 @@ protected SpineSceneDesc.Builder transform(Task task, IResource resource, SpineS
 
         return builder;
     }
+
+    @Override
+    public void build(Task task) throws CompileExceptionError, IOException {
+        super.build(task);
+
+        IResource testurec = null;
+        IResource spinejsonc = null;
+        for (IResource input: task.getInputs()) {
+            String path = input.getPath();
+            if (path.endsWith("texturesetc")) {
+                testurec = input;
+            }
+            else if (path.endsWith("spinejsonc")) {
+                spinejsonc = input;
+            }
+        }
+        try {
+            Spine.SPINE_LoadFileFromBuffer(spinejsonc.getContent(), spinejsonc.getPath(), testurec.getContent(), testurec.getPath());
```

Comment:
here we make an attempt to load spinemodel, if it wasn't loaded : throw compilation error